.github/workflows/template-ledger-epoch-snapshots.yml

```
@@ -30,6 +32,10 @@ jobs:
   snapshots:
     name: End-to-end snapshot tests
     runs-on: ubuntu-24.04
+    concurrency:
+      group: ${{ matrix.network.name == 'preprod' && 'preprod-e2e-transaction-submission' || format('ledger-epoch-snapshots-{0}-{1}', github.run_id, matrix.network.name) }}
+      cancel-in-progress: false
+      queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)